src/utils/UserId.ts

```diff
@@ -0,0 +1 @@
+export const USER_ID = 1591;
```

Comment:
Is this a `constant.ts`, not a `utils`)

Reply:
some mentors told us to move it to utils so idk